pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -107,6 +107,11 @@ func (admitter *VMICreateAdmitter) Admit(ar *admissionv1.AdmissionReview) *admis
 	}
 
 	var causes []metav1.StatusCause
+	clusterCfg := admitter.ClusterConfig.GetConfig()
+	if devCfg := clusterCfg.DeveloperConfiguration; devCfg != nil {
+		causes = append(causes, deprecation.ValidateFeatureGates(devCfg.FeatureGates, &vmi.Spec)...)
+	}
```

Comment:
nit: It seems this code repeats and the only thing that changes is the VMI spec "path", maybe you can export it to a helper.